python/activity_stream/activity_stream.py

```diff
@@ -102,7 +109,51 @@ def destroy(self):
         Should be called before the widget is closed
         """
         self._data_manager.destroy()
-        self._task_manager = None        
+        self._task_manager = None
+
+    ############################################################################
+    # properties
+
+    def _get_allow_screenshots(self):
+        """
+        Whether screenshots are being allowed.
+        """
+        return self._allow_screenshots
+
+    def _set_allow_screenshots(self, state):
+        """
+        Sets whether to allow screenshots.
+
+        :param state: True or False
+        """
+        self._allow_screenshots = bool(state)
+        self.ui.note_widget.allow_screenshots(self._allow_screenshots)
+
+    allow_screenshots = QtCore.Property(
```

Comment:
Looks like Sphinx pulls property docs from the method specified as the property's getter.